app/src/main/res/values/component_colors.xml

```diff
@@ -36,4 +36,12 @@
   <color name="component_color_admin_pin_activity_layout_background_color">@color/color_palette_background_color</color>
   <!-- Pin Password Activity -->
   <color name="component_color_pin_password_activity_layout_background_color">@color/color_palette_background_color</color>
+  <!-- HelpActivity -->
+  <color name="component_color_help_activity_background_color">@color/color_palette_dark_background_color</color>
+  <color name="component_color_help_activity_container_color">@color/color_palette_container_background_color</color>
+  <!-- FAQActivity -->
+  <color name="component_color_faq_activity_background_color">@color/color_palette_dark_background_color</color>
+  <color name="component_color_faq_activity_text_color">@color/color_palette_dark_text_color</color>
+  <color name="component_color_faq_container_background_color">@color/color_palette_container_background_color</color>
```

Comment:
component_color_faq_activity_container_background_color

Or you can directly use `component_color_faq_activity_background_color ` in whichever xml file its required

Reply:
Done